src/test/appForm.test.js

```diff
@@ -117,6 +179,45 @@ describe("App Form", function () {
         FormActions.update("appId", "/app-1");
       });
 
+      describe("the cpus field", function () {
+
+        it("updates correctly", function (done) {
+          AppFormStore.once(FormEvents.CHANGE, function () {
+            expectAsync(function () {
+
+              expect(AppFormStore.fields.cpus).to.equal(1.1);
+            }, done);
+          });
+
+          FormActions.update("cpus", 1.1);
+        });
+      });
+
+      describe("the disk field", function () {
+        it("updates correctly", function (done) {
+          AppFormStore.once(FormEvents.CHANGE, function () {
+            expectAsync(function () {
+              expect(AppFormStore.fields.disk).to.equal(256);
+            }, done);
+          });
+
+          FormActions.update("disk", 256);
+        });
+      });
+
+      describe("the cmd field", function () {
```

Comment:
Compare line 84 with line 208.
Also there mem, disk, etc. seems to be doubled too.